### ISSUE TITLE
All Flink jobs not finishing successfully

### ISSUE THREAD

**Issue by markus-h** [stratosphere/peel#31]
With the current Flink version all jobs that are running from peel are not finishing successfully.
The reason is that Flink does not support the printing of the plan from the CLI anymore.

The method 
```
override def isSuccessful = state.plnExitCode.getOrElse(-1) == 0 && state.runExitCode.getOrElse(-1) == 0
```
in FlinkExperiment will always return false.

A possible solution would be to change isSuccessful to only check for correct runExitCode and remove the check for plnExitCode.

**Comment by aalexandrov** [comment]
This is a known bug that does not effect the experiment lifecycle (@tammymendt managed to run experiment suites with this this week). I prefer to have a solution where we subclass the `Flink` system and pack the logic that varies between different systems abstract.

**Comment by aalexandrov** [comment]
Should be fixed with this 3686d328520a046e6ad96019462a9a75ebaef10d.
